addons/im_livechat/static/src/core/public_web/discuss_migration_0.js

```diff
@@ -0,0 +1,17 @@
+import { migrationHelpers } from "@mail/core/common/discuss_migration/discuss_migration_0";
+import { patch } from "@web/core/utils/patch";
+
+patch(migrationHelpers, {
+    discussAppCategoryLivechatDefaultLocalId: "DiscussAppCategory,im_livechat.category_default",
+});
+
+migrationHelpers.add("discuss_sidebar_category_folded_im_livechat.category_default", () => ({
+    key: migrationHelpers.makeRecordFieldLocalId(
+        migrationHelpers.discussAppCategoryLivechatDefaultLocalId,
+        "is_open"
+    ),
+    value: false,
+}));
+
+// discuss_sidebar_category_folded_(livechat_custom_id) is not migrated,
+// as we cannot know which local storage entry this maps unless we loop on whole local storage
```

Comment:
I still think we need to be doing it.
Correct API for migration should be able to handle this kind of case.

Going from one version to the next one should just iterate all keys present and know what to do for each of them.